### PR TITLE
Add a comment advising blurb-it if a news entry is missing

### PR DESCRIPTION
**Pull request by arhadthedev**
~*Since I have no possibility to test the addition, I make this PR draft until maintainers check it.*~

This PR complements `No news entry` status with a comment recommending https://blurb-it.herokuapp.com so a new contributor will know how to generate a NEWS entry. The comment is a verbatim copy of [the one by Serhiy Storchaka](https://github.com/python/cpython/pull/29947#issuecomment-987846284).

### CODE REVIEW

**Comment by brettcannon** [comment]
Without commenting on whether we would take this, the test suite allows you to test whether this would work (and we require 100% test coverage and so a test would be necessary).

**Comment by arhadthedev** [comment]
I added the test, could you approve the run please?

In addition I rephrased the message to mention the offline tool:

> To inform end users, a NEWS entry is necessary. See [Updating NEWS and What’s New in Python](https://devguide.python.org/committing/#updating-news-and-what-s-new-in-python) for more details.
>
> Please, add it using https://blurb-it.herokuapp.com site or [blurb PyPI package](https://pypi.org/project/blurb/); they both take care on names and placement.

If it can be further improved, I'll happily make edits.

**Comment by Mariatta** [comment]
Somehow the coverage bot is not triggered here. Not sure what's going on, but perhaps you need to rebase with the latest change.

I think we were trying to avoid posting too many bot messages :) If people find this message useful instead of too noisy, then perhaps it's ok.

**Comment by AlexWaygood** [comment]
>  If people find this message useful instead of too noisy, then perhaps it's ok.

I would have found this useful when I was new to contributing to CPython. I didn't learn about the blurb_it website until a few months after I'd started contributing. I actually didn't realise that the "Details" button in the CI check linked straight to blurb_it until a few weeks ago.

I also understand the concern about too many comments by bots, though. Ideally I guess I'd like to see the comment only triggered if it's a PR by a first-time contributor, but I guess that would add some complexity.

**Comment by Mariatta** [comment]
> I guess I'd like to see the comment only triggered if it's a PR by a first-time contributor, but I guess that would add some complexity.

The `pull_request` webhook payload includes the `"author_association"` info. The values could be: `"OWNER"`, `"MEMBER"`, `"CONTRIBUTOR"`, or `"NONE"`.
In this case, we can issue the bot comment if the `"author_association"` field is `"NONE"`.


Example real webhook payload data:

<details>

```

{
  "action": "opened",
  "number": 91478,
  "pull_request": {
    "url": "https://api.github.com/repos/python/cpython/pulls/91478",
    "id": 907417240,
    "node_id": "PR_kwDOBN0Z8c42FhaY",
    "html_url": "https://github.com/python/cpython/pull/91478",
    "diff_url": "https://github.com/python/cpython/pull/91478.diff",
    "patch_url": "https://github.com/python/cpython/pull/91478.patch",
    "issue_url": "https://api.github.com/repos/python/cpython/issues/91478",
    "number": 91478,
    "state": "open",
    "locked": false,
    "title": "updated tutorial example code output",
    "user": {
      "login": "alex-krav",
      "id": 7326297,
      "node_id": "MDQ6VXNlcjczMjYyOTc=",
      "avatar_url": "https://avatars.githubusercontent.com/u/7326297?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/alex-krav",
      "html_url": "https://github.com/alex-krav",
      "followers_url": "https://api.github.com/users/alex-krav/followers",
      "following_url": "https://api.github.com/users/alex-krav/following{/other_user}",
      "gists_url": "https://api.github.com/users/alex-krav/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/alex-krav/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/alex-krav/subscriptions",
      "organizations_url": "https://api.github.com/users/alex-krav/orgs",
      "repos_url": "https://api.github.com/users/alex-krav/repos",
      "events_url": "https://api.github.com/users/alex-krav/events{/privacy}",
      "received_events_url": "https://api.github.com/users/alex-krav/received_events",
      "type": "User",
      "site_admin": false
    },
    "body": "<!--\r\nThanks for your contribution!\r\nPlease read this comment in its entirety. It's quite important.\r\n\r\n# Pull Request title\r\n\r\nIt should be in the following format:\r\n\r\n```\r\ngh-NNNNN: Summary of the changes made\r\n```\r\n\r\nWhere: gh-NNNNN refers to the GitHub issue number.\r\n\r\nMost PRs will require an issue number. Trivial changes, like fixing a typo, do not need an issue.\r\n\r\n# Backport Pull Request title\r\n\r\nIf this is a backport PR (PR made against branches other than `main`),\r\nplease ensure that the PR title is in the following format:\r\n\r\n```\r\n[X.Y] <title from the original PR> (GH-NNNN)\r\n```\r\n\r\nWhere: [X.Y] is the branch name, e.g. [3.6].\r\n\r\nGH-NNNN refers to the PR number from `main`.\r\n\r\n-->\r\n",
    "created_at": "2022-04-12T09:45:57Z",
    "updated_at": "2022-04-12T09:45:57Z",
    "closed_at": null,
    "merged_at": null,
    "merge_commit_sha": null,
    "assignee": null,
    "assignees": [

    ],
    "requested_reviewers": [

    ],
    "requested_teams": [

    ],
    "labels": [

    ],
    "milestone": null,
    "draft": false,
    "commits_url": "https://api.github.com/repos/python/cpython/pulls/91478/commits",
    "review_comments_url": "https://api.github.com/repos/python/cpython/pulls/91478/comments",
    "review_comment_url": "https://api.github.com/repos/python/cpython/pulls/comments{/number}",
    "comments_url": "https://api.github.com/repos/python/cpython/issues/91478/comments",
    "statuses_url": "https://api.github.com/repos/python/cpython/statuses/4b214f4208c69680a0381c2ca55fd21aab1b7cc7",
    "head": {
      "label": "alex-krav:docs-update",
      "ref": "docs-update",
      "sha": "4b214f4208c69680a0381c2ca55fd21aab1b7cc7",
      "user": {
        "login": "alex-krav",
        "id": 7326297,
        "node_id": "MDQ6VXNlcjczMjYyOTc=",
        "avatar_url": "https://avatars.githubusercontent.com/u/7326297?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/alex-krav",
        "html_url": "https://github.com/alex-krav",
        "followers_url": "https://api.github.com/users/alex-krav/followers",
        "following_url": "https://api.github.com/users/alex-krav/following{/other_user}",
        "gists_url": "https://api.github.com/users/alex-krav/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/alex-krav/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/alex-krav/subscriptions",
        "organizations_url": "https://api.github.com/users/alex-krav/orgs",
        "repos_url": "https://api.github.com/users/alex-krav/repos",
        "events_url": "https://api.github.com/users/alex-krav/events{/privacy}",
        "received_events_url": "https://api.github.com/users/alex-krav/received_events",
        "type": "User",
        "site_admin": false
      },
      "repo": {
        "id": 479493720,
        "node_id": "R_kgDOHJR-WA",
        "name": "cpython",
        "full_name": "alex-krav/cpython",
        "private": false,
        "owner": {
          "login": "alex-krav",
          "id": 7326297,
          "node_id": "MDQ6VXNlcjczMjYyOTc=",
          "avatar_url": "https://avatars.githubusercontent.com/u/7326297?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/alex-krav",
          "html_url": "https://github.com/alex-krav",
          "followers_url": "https://api.github.com/users/alex-krav/followers",
          "following_url": "https://api.github.com/users/alex-krav/following{/other_user}",
          "gists_url": "https://api.github.com/users/alex-krav/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/alex-krav/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/alex-krav/subscriptions",
          "organizations_url": "https://api.github.com/users/alex-krav/orgs",
          "repos_url": "https://api.github.com/users/alex-krav/repos",
          "events_url": "https://api.github.com/users/alex-krav/events{/privacy}",
          "received_events_url": "https://api.github.com/users/alex-krav/received_events",
          "type": "User",
          "site_admin": false
        },
        "html_url": "https://github.com/alex-krav/cpython",
        "description": "The Python programming language",
        "fork": true,
        "url": "https://api.github.com/repos/alex-krav/cpython",
        "forks_url": "https://api.github.com/repos/alex-krav/cpython/forks",
        "keys_url": "https://api.github.com/repos/alex-krav/cpython/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/alex-krav/cpython/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/alex-krav/cpython/teams",
        "hooks_url": "https://api.github.com/repos/alex-krav/cpython/hooks",
        "issue_events_url": "https://api.github.com/repos/alex-krav/cpython/issues/events{/number}",
        "events_url": "https://api.github.com/repos/alex-krav/cpython/events",
        "assignees_url": "https://api.github.com/repos/alex-krav/cpython/assignees{/user}",
        "branches_url": "https://api.github.com/repos/alex-krav/cpython/branches{/branch}",
        "tags_url": "https://api.github.com/repos/alex-krav/cpython/tags",
        "blobs_url": "https://api.github.com/repos/alex-krav/cpython/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/alex-krav/cpython/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/alex-krav/cpython/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/alex-krav/cpython/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/alex-krav/cpython/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/alex-krav/cpython/languages",
        "stargazers_url": "https://api.github.com/repos/alex-krav/cpython/stargazers",
        "contributors_url": "https://api.github.com/repos/alex-krav/cpython/contributors",
        "subscribers_url": "https://api.github.com/repos/alex-krav/cpython/subscribers",
        "subscription_url": "https://api.github.com/repos/alex-krav/cpython/subscription",
        "commits_url": "https://api.github.com/repos/alex-krav/cpython/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/alex-krav/cpython/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/alex-krav/cpython/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/alex-krav/cpython/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/alex-krav/cpython/contents/{+path}",
        "compare_url": "https://api.github.com/repos/alex-krav/cpython/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/alex-krav/cpython/merges",
        "archive_url": "https://api.github.com/repos/alex-krav/cpython/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/alex-krav/cpython/downloads",
        "issues_url": "https://api.github.com/repos/alex-krav/cpython/issues{/number}",
        "pulls_url": "https://api.github.com/repos/alex-krav/cpython/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/alex-krav/cpython/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/alex-krav/cpython/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/alex-krav/cpython/labels{/name}",
        "releases_url": "https://api.github.com/repos/alex-krav/cpython/releases{/id}",
        "deployments_url": "https://api.github.com/repos/alex-krav/cpython/deployments",
        "created_at": "2022-04-08T18:11:34Z",
        "updated_at": "2022-04-08T17:03:13Z",
        "pushed_at": "2022-04-12T09:43:06Z",
        "git_url": "git://github.com/alex-krav/cpython.git",
        "ssh_url": "git@github.com:alex-krav/cpython.git",
        "clone_url": "https://github.com/alex-krav/cpython.git",
        "svn_url": "https://github.com/alex-krav/cpython",
        "homepage": "https://www.python.org/",
        "size": 466783,
        "stargazers_count": 0,
        "watchers_count": 0,
        "language": null,
        "has_issues": false,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": false,
        "has_pages": false,
        "forks_count": 0,
        "mirror_url": null,
        "archived": false,
        "disabled": false,
        "open_issues_count": 0,
        "license": {
          "key": "other",
          "name": "Other",
          "spdx_id": "NOASSERTION",
          "url": null,
          "node_id": "MDc6TGljZW5zZTA="
        },
        "allow_forking": true,
        "is_template": false,
        "topics": [

        ],
        "visibility": "public",
        "forks": 0,
        "open_issues": 0,
        "watchers": 0,
        "default_branch": "main",
        "allow_squash_merge": true,
        "allow_merge_commit": true,
        "allow_rebase_merge": true,
        "allow_auto_merge": false,
        "delete_branch_on_merge": false,
        "allow_update_branch": false
      }
    },
    "base": {
      "label": "python:main",
      "ref": "main",
      "sha": "f33e2c87a83917b5139d97fd8ef7cba7223ebef5",
      "user": {
        "login": "python",
        "id": 1525981,
        "node_id": "MDEyOk9yZ2FuaXphdGlvbjE1MjU5ODE=",
        "avatar_url": "https://avatars.githubusercontent.com/u/1525981?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/python",
        "html_url": "https://github.com/python",
        "followers_url": "https://api.github.com/users/python/followers",
        "following_url": "https://api.github.com/users/python/following{/other_user}",
        "gists_url": "https://api.github.com/users/python/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/python/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/python/subscriptions",
        "organizations_url": "https://api.github.com/users/python/orgs",
        "repos_url": "https://api.github.com/users/python/repos",
        "events_url": "https://api.github.com/users/python/events{/privacy}",
        "received_events_url": "https://api.github.com/users/python/received_events",
        "type": "Organization",
        "site_admin": false
      },
      "repo": {
        "id": 81598961,
        "node_id": "MDEwOlJlcG9zaXRvcnk4MTU5ODk2MQ==",
        "name": "cpython",
        "full_name": "python/cpython",
        "private": false,
        "owner": {
          "login": "python",
          "id": 1525981,
          "node_id": "MDEyOk9yZ2FuaXphdGlvbjE1MjU5ODE=",
          "avatar_url": "https://avatars.githubusercontent.com/u/1525981?v=4",
          "gravatar_id": "",
          "url": "https://api.github.com/users/python",
          "html_url": "https://github.com/python",
          "followers_url": "https://api.github.com/users/python/followers",
          "following_url": "https://api.github.com/users/python/following{/other_user}",
          "gists_url": "https://api.github.com/users/python/gists{/gist_id}",
          "starred_url": "https://api.github.com/users/python/starred{/owner}{/repo}",
          "subscriptions_url": "https://api.github.com/users/python/subscriptions",
          "organizations_url": "https://api.github.com/users/python/orgs",
          "repos_url": "https://api.github.com/users/python/repos",
          "events_url": "https://api.github.com/users/python/events{/privacy}",
          "received_events_url": "https://api.github.com/users/python/received_events",
          "type": "Organization",
          "site_admin": false
        },
        "html_url": "https://github.com/python/cpython",
        "description": "The Python programming language",
        "fork": false,
        "url": "https://api.github.com/repos/python/cpython",
        "forks_url": "https://api.github.com/repos/python/cpython/forks",
        "keys_url": "https://api.github.com/repos/python/cpython/keys{/key_id}",
        "collaborators_url": "https://api.github.com/repos/python/cpython/collaborators{/collaborator}",
        "teams_url": "https://api.github.com/repos/python/cpython/teams",
        "hooks_url": "https://api.github.com/repos/python/cpython/hooks",
        "issue_events_url": "https://api.github.com/repos/python/cpython/issues/events{/number}",
        "events_url": "https://api.github.com/repos/python/cpython/events",
        "assignees_url": "https://api.github.com/repos/python/cpython/assignees{/user}",
        "branches_url": "https://api.github.com/repos/python/cpython/branches{/branch}",
        "tags_url": "https://api.github.com/repos/python/cpython/tags",
        "blobs_url": "https://api.github.com/repos/python/cpython/git/blobs{/sha}",
        "git_tags_url": "https://api.github.com/repos/python/cpython/git/tags{/sha}",
        "git_refs_url": "https://api.github.com/repos/python/cpython/git/refs{/sha}",
        "trees_url": "https://api.github.com/repos/python/cpython/git/trees{/sha}",
        "statuses_url": "https://api.github.com/repos/python/cpython/statuses/{sha}",
        "languages_url": "https://api.github.com/repos/python/cpython/languages",
        "stargazers_url": "https://api.github.com/repos/python/cpython/stargazers",
        "contributors_url": "https://api.github.com/repos/python/cpython/contributors",
        "subscribers_url": "https://api.github.com/repos/python/cpython/subscribers",
        "subscription_url": "https://api.github.com/repos/python/cpython/subscription",
        "commits_url": "https://api.github.com/repos/python/cpython/commits{/sha}",
        "git_commits_url": "https://api.github.com/repos/python/cpython/git/commits{/sha}",
        "comments_url": "https://api.github.com/repos/python/cpython/comments{/number}",
        "issue_comment_url": "https://api.github.com/repos/python/cpython/issues/comments{/number}",
        "contents_url": "https://api.github.com/repos/python/cpython/contents/{+path}",
        "compare_url": "https://api.github.com/repos/python/cpython/compare/{base}...{head}",
        "merges_url": "https://api.github.com/repos/python/cpython/merges",
        "archive_url": "https://api.github.com/repos/python/cpython/{archive_format}{/ref}",
        "downloads_url": "https://api.github.com/repos/python/cpython/downloads",
        "issues_url": "https://api.github.com/repos/python/cpython/issues{/number}",
        "pulls_url": "https://api.github.com/repos/python/cpython/pulls{/number}",
        "milestones_url": "https://api.github.com/repos/python/cpython/milestones{/number}",
        "notifications_url": "https://api.github.com/repos/python/cpython/notifications{?since,all,participating}",
        "labels_url": "https://api.github.com/repos/python/cpython/labels{/name}",
        "releases_url": "https://api.github.com/repos/python/cpython/releases{/id}",
        "deployments_url": "https://api.github.com/repos/python/cpython/deployments",
        "created_at": "2017-02-10T19:23:51Z",
        "updated_at": "2022-04-12T09:43:41Z",
        "pushed_at": "2022-04-12T09:30:06Z",
        "git_url": "git://github.com/python/cpython.git",
        "ssh_url": "git@github.com:python/cpython.git",
        "clone_url": "https://github.com/python/cpython.git",
        "svn_url": "https://github.com/python/cpython",
        "homepage": "https://www.python.org/",
        "size": 467297,
        "stargazers_count": 43970,
        "watchers_count": 43970,
        "language": "Python",
        "has_issues": true,
        "has_projects": true,
        "has_downloads": true,
        "has_wiki": false,
        "has_pages": false,
        "forks_count": 22735,
        "mirror_url": null,
        "archived": false,
        "disabled": false,
        "open_issues_count": 8654,
        "license": {
          "key": "other",
          "name": "Other",
          "spdx_id": "NOASSERTION",
          "url": null,
          "node_id": "MDc6TGljZW5zZTA="
        },
        "allow_forking": true,
        "is_template": false,
        "topics": [

        ],
        "visibility": "public",
        "forks": 22735,
        "open_issues": 8654,
        "watchers": 43970,
        "default_branch": "main",
        "allow_squash_merge": true,
        "allow_merge_commit": false,
        "allow_rebase_merge": false,
        "allow_auto_merge": false,
        "delete_branch_on_merge": true,
        "allow_update_branch": true
      }
    },
    "_links": {
      "self": {
        "href": "https://api.github.com/repos/python/cpython/pulls/91478"
      },
      "html": {
        "href": "https://github.com/python/cpython/pull/91478"
      },
      "issue": {
        "href": "https://api.github.com/repos/python/cpython/issues/91478"
      },
      "comments": {
        "href": "https://api.github.com/repos/python/cpython/issues/91478/comments"
      },
      "review_comments": {
        "href": "https://api.github.com/repos/python/cpython/pulls/91478/comments"
      },
      "review_comment": {
        "href": "https://api.github.com/repos/python/cpython/pulls/comments{/number}"
      },
      "commits": {
        "href": "https://api.github.com/repos/python/cpython/pulls/91478/commits"
      },
      "statuses": {
        "href": "https://api.github.com/repos/python/cpython/statuses/4b214f4208c69680a0381c2ca55fd21aab1b7cc7"
      }
    },
    "author_association": "NONE",
    "auto_merge": null,
    "active_lock_reason": null,
    "merged": false,
    "mergeable": null,
    "rebaseable": null,
    "mergeable_state": "unknown",
    "merged_by": null,
    "comments": 0,
    "review_comments": 0,
    "maintainer_can_modify": true,
    "commits": 1,
    "additions": 6,
    "deletions": 6,
    "changed_files": 5
  },
  "repository": {
    "id": 81598961,
    "node_id": "MDEwOlJlcG9zaXRvcnk4MTU5ODk2MQ==",
    "name": "cpython",
    "full_name": "python/cpython",
    "private": false,
    "owner": {
      "login": "python",
      "id": 1525981,
      "node_id": "MDEyOk9yZ2FuaXphdGlvbjE1MjU5ODE=",
      "avatar_url": "https://avatars.githubusercontent.com/u/1525981?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/python",
      "html_url": "https://github.com/python",
      "followers_url": "https://api.github.com/users/python/followers",
      "following_url": "https://api.github.com/users/python/following{/other_user}",
      "gists_url": "https://api.github.com/users/python/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/python/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/python/subscriptions",
      "organizations_url": "https://api.github.com/users/python/orgs",
      "repos_url": "https://api.github.com/users/python/repos",
      "events_url": "https://api.github.com/users/python/events{/privacy}",
      "received_events_url": "https://api.github.com/users/python/received_events",
      "type": "Organization",
      "site_admin": false
    },
    "html_url": "https://github.com/python/cpython",
    "description": "The Python programming language",
    "fork": false,
    "url": "https://api.github.com/repos/python/cpython",
    "forks_url": "https://api.github.com/repos/python/cpython/forks",
    "keys_url": "https://api.github.com/repos/python/cpython/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/python/cpython/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/python/cpython/teams",
    "hooks_url": "https://api.github.com/repos/python/cpython/hooks",
    "issue_events_url": "https://api.github.com/repos/python/cpython/issues/events{/number}",
    "events_url": "https://api.github.com/repos/python/cpython/events",
    "assignees_url": "https://api.github.com/repos/python/cpython/assignees{/user}",
    "branches_url": "https://api.github.com/repos/python/cpython/branches{/branch}",
    "tags_url": "https://api.github.com/repos/python/cpython/tags",
    "blobs_url": "https://api.github.com/repos/python/cpython/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/python/cpython/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/python/cpython/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/python/cpython/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/python/cpython/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/python/cpython/languages",
    "stargazers_url": "https://api.github.com/repos/python/cpython/stargazers",
    "contributors_url": "https://api.github.com/repos/python/cpython/contributors",
    "subscribers_url": "https://api.github.com/repos/python/cpython/subscribers",
    "subscription_url": "https://api.github.com/repos/python/cpython/subscription",
    "commits_url": "https://api.github.com/repos/python/cpython/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/python/cpython/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/python/cpython/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/python/cpython/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/python/cpython/contents/{+path}",
    "compare_url": "https://api.github.com/repos/python/cpython/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/python/cpython/merges",
    "archive_url": "https://api.github.com/repos/python/cpython/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/python/cpython/downloads",
    "issues_url": "https://api.github.com/repos/python/cpython/issues{/number}",
    "pulls_url": "https://api.github.com/repos/python/cpython/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/python/cpython/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/python/cpython/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/python/cpython/labels{/name}",
    "releases_url": "https://api.github.com/repos/python/cpython/releases{/id}",
    "deployments_url": "https://api.github.com/repos/python/cpython/deployments",
    "created_at": "2017-02-10T19:23:51Z",
    "updated_at": "2022-04-12T09:43:41Z",
    "pushed_at": "2022-04-12T09:30:06Z",
    "git_url": "git://github.com/python/cpython.git",
    "ssh_url": "git@github.com:python/cpython.git",
    "clone_url": "https://github.com/python/cpython.git",
    "svn_url": "https://github.com/python/cpython",
    "homepage": "https://www.python.org/",
    "size": 467297,
    "stargazers_count": 43970,
    "watchers_count": 43970,
    "language": "Python",
    "has_issues": true,
    "has_projects": true,
    "has_downloads": true,
    "has_wiki": false,
    "has_pages": false,
    "forks_count": 22735,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 8654,
    "license": {
      "key": "other",
      "name": "Other",
      "spdx_id": "NOASSERTION",
      "url": null,
      "node_id": "MDc6TGljZW5zZTA="
    },
    "allow_forking": true,
    "is_template": false,
    "topics": [

    ],
    "visibility": "public",
    "forks": 22735,
    "open_issues": 8654,
    "watchers": 43970,
    "default_branch": "main"
  },
  "organization": {
    "login": "python",
    "id": 1525981,
    "node_id": "MDEyOk9yZ2FuaXphdGlvbjE1MjU5ODE=",
    "url": "https://api.github.com/orgs/python",
    "repos_url": "https://api.github.com/orgs/python/repos",
    "events_url": "https://api.github.com/orgs/python/events",
    "hooks_url": "https://api.github.com/orgs/python/hooks",
    "issues_url": "https://api.github.com/orgs/python/issues",
    "members_url": "https://api.github.com/orgs/python/members{/member}",
    "public_members_url": "https://api.github.com/orgs/python/public_members{/member}",
    "avatar_url": "https://avatars.githubusercontent.com/u/1525981?v=4",
    "description": "Repositories related to the Python Programming language"
  },
  "sender": {
    "login": "alex-krav",
    "id": 7326297,
    "node_id": "MDQ6VXNlcjczMjYyOTc=",
    "avatar_url": "https://avatars.githubusercontent.com/u/7326297?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/alex-krav",
    "html_url": "https://github.com/alex-krav",
    "followers_url": "https://api.github.com/users/alex-krav/followers",
    "following_url": "https://api.github.com/users/alex-krav/following{/other_user}",
    "gists_url": "https://api.github.com/users/alex-krav/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/alex-krav/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/alex-krav/subscriptions",
    "organizations_url": "https://api.github.com/users/alex-krav/orgs",
    "repos_url": "https://api.github.com/users/alex-krav/repos",
    "events_url": "https://api.github.com/users/alex-krav/events{/privacy}",
    "received_events_url": "https://api.github.com/users/alex-krav/received_events",
    "type": "User",
    "site_admin": false
  }
}

```

</details>

File: bedevere/news.py [hunk 1]
```diff
@@ -52,6 +60,8 @@ async def check_news(gh, pull_request, files=None):
         if util.skip("news", issue):
             status = SKIP_LABEL_STATUS
         else:
+            await gh.post(pull_request['issue_comment_url'],
```
Comment (Mariatta) [inline bedevere/news.py:63]: Add before this:

```
if pull_request['author_association'] == 'NONE':
```

Comment (Mariatta) [reply bedevere/news.py:63]: `pull_request['issue_comment_url']` is not the right POST url.

API doc: https://docs.github.com/en/rest/reference/issues#create-an-issue-comment

The path is `/repos/{owner}/{repo}/issues/{issue_number}/comments`, so this should look like:
```suggestion
            await gh.post(f"{pull_request['issue_url']}/comments",
```

**Review by Mariatta — CHANGES_REQUESTED** [review]
The `issue_comment_url` you're using is not correct.
Also, let's check if the user is a first-time contributor by checking the `"author_association"` field.

**Comment by arhadthedev** [comment]
@ambv The status went green because I've resigned the CLA.

File: bedevere/news.py [hunk 2]
```diff
@@ -26,6 +27,13 @@
 SKIP_LABEL_STATUS = create_status(util.StatusState.SUCCESS,
                                   description='"skip news" label found')
 
+HELP = f"""\
+To inform end users, a NEWS entry is necessary. See [Updating NEWS and \
+What’s New in Python](https://devguide.python.org/committing/#updating-news-\
+and-what-s-new-in-python) for more details.
+
+Please, add it using {BLURB_IT_URL} site or [blurb PyPI \
+package]({BLURB_PYPI_URL}); they both take care on names and placement."""
```
Comment (ambv) [inline bedevere/news.py:36]:
```suggestion
Every change to Python [requires a NEWS entry]\
(https://devguide.python.org/committing/#updating-news-and-what-s-new-in-python).

Please, add it using the [blurb_it]({BLURB_IT_URL}) Web app or the [blurb]\
({BLURB_PYPI_URL}) command-line tool."""
```

**Comment by AlexWaygood** [comment]
Thanks @arhadthedev, this is great! 😀 https://github.com/python/cpython/pull/47246#issuecomment-1100101716

**Pull request merged**